cookbooks/aws-parallelcluster-platform/files/ami_cleanup.sh

```diff
@@ -21,8 +23,10 @@ if [ "${ID}${VERSION_ID}" == "centos7" ]; then
 fi
 
 # Clean resolv.conf if it's not managed by system
```

Comment:
could you please align this comment to the current strategy and put a short explanation, so that we will not forget i future?